### PR TITLE
fix incomplete merged from PR 43 (#60)

### PR DESCRIPTION
**Pull request by nat236919**
## Related issue

- https://github.com/t0xic0der/sanctuary-zero/issues/60

## Fixes

- Restore promp_continuation for long input
- Remove a duplicate function

## Screenshots (after fixes)

image

image

### CODE REVIEW

**Comment by t0xic0der** [comment]
Let me get into reviewing it.

File: cnew.py [hunk 1]
```diff
@@ -51,7 +51,10 @@ async def producer_handler(cphrsuit, websocket, username, chatroom, servaddr):
         footelem = HTML("<b>[" + chatroom + "]</b>" + " " + username.strip() + " - Sanctuary ZERO v04092020 running on '" + servaddr + "' - Hit Ctrl+C to EXIT")
         while True:
             with patch_stdout():
-                mesgtext = await sess.prompt_async(lambda:"[" + obtntime() + "] " + formusnm(str(username)) + " > ", bottom_toolbar=footelem, validator=emtyfind(), refresh_interval=0.5)
+                mesgtext = await sess.prompt_async(lambda:"[" + obtntime() + "] " + formusnm(str(username)) + " > ", bottom_toolbar=footelem,
+                                                                                                                     validator=emtyfind(),
+                                                                                                                     refresh_interval=0.5,
+                                                                                                                     prompt_continuation=lambda width, line_number, is_soft_wrap: ' ' * width)
```
Comment (t0xic0der) [inline cnew.py:57]: Yes, the latest commit on `master` does not seem to have the `prompt_continuation`. Thanks for adding it back.

File: main.py [hunk 2]
```diff
@@ -32,22 +32,6 @@ async def notify_mesej(message):
...
-        indent_text += ' '
-    for i in range(0, message_width, width):
-        if i > 0:
-            wrapped_message += indent_text
-        wrapped_message += message[i : i + width]
-        if i < message_width - width:
-            wrapped_message += '\n'
-    return wrapped_message
-
-
```
Comment (t0xic0der) [inline main.py:50]: With `helper_display` of `utils`, this function is no longer needed.

**Review by t0xic0der — APPROVED** [review]
Looks good to me.

**Comment by nat236919** [comment]
> Looks good to me.

Thank you.

**Pull request merged**